### ISSUE TITLE
no matching function for call to

### ISSUE THREAD

**Issue by lvl3rr0r** [voltasonnet/RFID_Sniffer#1]
good work but
i got error while verfying 
can you help me  ?

exit status 1
no matching function for call to 'MFRC522::PCD_SetRegisterBitMask(PCD_Register, int)'

**Comment by voltasonnet** [comment]
I have not worked on this in a while but I will try my best to help! You are getting this error when trying to compile the program in the Arduino IDE? Do you have the MFRC522 library installed?

**Comment by lvl3rr0r** [comment]
i can compile by version 1.2.0 MFRC522 library but the output is endless
FIFO Data Reg At Offset:  ==>  0x80

**Comment by voltasonnet** [comment]
Yeah, it is coming back to me a little. I read some of the published papers on how two graduate students were able to reverse engineer the protocol the MiFare card uses and I noticed two things, the transmissions between the card and the reader were too fast for this Arduino to catch them all, and that I was seeing the same pattern over and over again. This lead me to compare the known values such as card select and other to my output and was able to piece together some information. It has to be really close to the card and the reader/phone, even increasing the sensitivity to its max value did not help much. It was proof of concept, I had a translator to get the Register info and get some useful data out but I guess I never pushed that. I am sure it is long gone by now. If you want to continue, I would recommend reading those papers (Hacking the MiFare was the name of one if IIRC), look at the sample communication table and start deciphering it by hand, build off of that.  I remember I was able to get a complete UID out at some point during this projects life cycle.
